llvm/docs/CMake.rst

```diff
@@ -832,9 +832,15 @@ enabled sub-projects. Nearly all of these variable names begin with
 **LLVM_TARGETS_TO_BUILD**:STRING
   Semicolon-separated list of targets to build, or *all* for building all
   targets. Case-sensitive. Defaults to *all*. Example:
-  ``-DLLVM_TARGETS_TO_BUILD="X86;PowerPC"``.
+  ``-DLLVM_TARGETS_TO_BUILD="X86;PowerPC;host"``.
   The full list, as of March 2023, is:
-  ``AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SystemZ;VE;WebAssembly;X86;XCore``
+  ``AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SystemZ;VE;WebAssembly;X86;XCore;host;Native``
```

Comment:
Listing additional options is a good idea, but I don't think they should be included in this list of targets, or the example above (esp. as for the majority of people, X86 is also host).

Reply:
Done.